src/test/java/unicash/logic/commands/AddTransactionCommandTest.java

```diff
@@ -59,7 +59,7 @@ public void equals() {
         Transaction nus = new TransactionBuilder().withName("Nus").build();
         Transaction intern = new TransactionBuilder().withName("Intern").build();
         AddTransactionCommand addNusCommand = new AddTransactionCommand(nus);
-        AddTransactionCommand addBobCommand = new AddTransactionCommand(intern);
+        AddTransactionCommand addInternCommand = new AddTransactionCommand(intern);
```

Comment:
Could we use the pre-built `NUS` and `INTERN` global instances? So there's no accidental changes that are made that aren't being captured

Reply:
Solved! Thanks for pointing this out